src/storage/mutate/AddEdgesProcessor.cpp

```diff
@@ -135,13 +133,15 @@ AddEdgesProcessor::addEdges(PartitionID partId,
      *
      * Ultimately, kv(part1_src1_edgeType1_rank1_dst1 , v4) . It's just what I need.
      */
-    std::map<std::string, std::string> newEdges;
+    std::unordered_map<std::string, std::string> newEdges;
```

Comment:
std::unordered_map<std::string, std::string> newEdges(edges.begin(), edges.end());

Reply:
Because the value of the same key needs to be updated later